scripts/create_datacheck_tickets.pl

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env perl
+
+=head1 LICENSE
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+=cut
+
+=head2 DESCRIPTION
+
+This script parses the output from Ensembl's Datachecks and creates JIRA tickets
+for each failure.
+
+=head1 SYNOPSIS
+
+perl create_datacheck_tickets.pl [options]
+
+=head1 OPTIONS
+
+=over 8
+
+=item B<-i[nput]> <datacheck_TAP_file>
+
+Mandatory. Path to a datacheck output TAP file.
+
+=item B<-r[elease]> <release>
+
+Mandatory. Ensembl release version.
+
+=item B<-p[roject]> <JIRA_project>
+
+Mandatory. JIRA project name, e.g. 'ENSPROD'.
+
+=item B<-user> <JIRA_user>
+
+Optional. JIRA username. If not given, uses environment variable $USER 
+as default.
+
+=item B<-components> <>
+
+Optional. JIRA component tags. Multiple values can be given as separate 
+-components parameters, or as a single comma-separated string.
+
+=item B<-dry_run>, B<-dry-run>
+
+In dry-run mode, the JIRA tickets will not be submitted to the JIRA
+server. By default, dry-run mode is off.
+
+=item B<-h[elp]>
+
+Print usage information.
+
+=back
+
+=cut
+
+
+use warnings;
+use strict;
+
+use Cwd 'abs_path';
+use File::Basename;
+use Getopt::Long;
+use Pod::Usage;
+use POSIX;
+
+use Bio::EnsEMBL::Compara::Utils::JIRA;
+
+use Data::Dumper;
+
+my ( $dc_file, $release, $project, $user, @components, $dry_run, $help );
+$dry_run = 0;
+$help    = 0;
+GetOptions(
+    "i|input=s"       => \$dc_file,
+    "r|release=s"     => \$release,
+    "p|project=s"     => \$project,
+    "user=s",         => \$user,
+    "components:s"    => \@components,
+    'dry_run|dry-run' => \$dry_run,
+    "h|help"          => \$help,
+);
+pod2usage(1) if $help;
+die "Cannot find $dc_file - file does not exist" unless -e $dc_file;
+die pod2usage(1) if (! ($dc_file && $release && $project));
+# Get file absolute path and basename
+my $dc_abs_path = abs_path($dc_file);
+my $dc_basename = fileparse($dc_abs_path, qr{\.[a-zA-Z0-9_]+$});
+# Get timestamp that will be included in the summary of each JIRA ticket
+my $timestamp = strftime("%d-%m-%Y %H:%M:%S", localtime time);
+# Get a new Utils::JIRA object to create the tickets for the given division and
+# release
+my $jira_adaptor = Bio::EnsEMBL::Compara::Utils::JIRA->new(
+    -USER     => $user,
```

Comment:
This instantiation fails, because there is a hard-coded list of acceptable usernames in Bio::EnsEMBL::Compara::Utils::JIRA, line 329

Reply:
This has been addressed already in our `master` branch.